### ISSUE TITLE
Steam doesn't scale it's window, but shows that the scale switch is on.

### ISSUE THREAD

**Issue by Inkrm** [ValveSoftware/steam-for-linux#9958]
#### Your system information

* Steam client version (build number or date): 1691193224
* Distribution (e.g. Ubuntu): Debian 12, KDE Plasma 5.27.5
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes
* Steam Logs: [generate by running this command in a terminal `tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs`]
* GPU: Nvidia

```
Steam logs:➜  ~ tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs
tar: Removing leading `/' from member names
/home/inkrm/.steam/steam/logs/
/home/inkrm/.steam/steam/logs/systemdockmanager.txt
/home/inkrm/.steam/steam/logs/timedtrial_log.txt
/home/inkrm/.steam/steam/logs/systemaudiomanager.txt
/home/inkrm/.steam/steam/logs/bootstrap_log.txt
/home/inkrm/.steam/steam/logs/transport_client.txt
/home/inkrm/.steam/steam/logs/steamwebhelper.log
/home/inkrm/.steam/steam/logs/cef_log.txt
/home/inkrm/.steam/steam/logs/console_log.txt
/home/inkrm/.steam/steam/logs/transport_steamui.txt
/home/inkrm/.steam/steam/logs/remote_connections.txt
/home/inkrm/.steam/steam/logs/cloud_log.txt
/home/inkrm/.steam/steam/logs/client_networkmanager.txt
/home/inkrm/.steam/steam/logs/appinfo_log.txt
/home/inkrm/.steam/steam/logs/content_log.txt
/home/inkrm/.steam/steam/logs/systemperfmanager.txt
/home/inkrm/.steam/steam/logs/configstore_log.txt
/home/inkrm/.steam/steam/logs/webhelper.txt
/home/inkrm/.steam/steam/logs/compat_log.txt
/home/inkrm/.steam/steam/logs/parental_log.txt
/home/inkrm/.steam/steam/logs/webhelper.previous.txt
/home/inkrm/.steam/steam/logs/steamui_system.txt
/home/inkrm/.steam/steam/logs/controller.txt
/home/inkrm/.steam/steam/logs/controller_ui.txt
/home/inkrm/.steam/steam/logs/shader_log.txt
/home/inkrm/.steam/steam/logs/systemdisplaymanager.txt
/home/inkrm/.steam/steam/logs/workshop_log.txt
/home/inkrm/.steam/steam/logs/systemmanager.txt
/home/inkrm/.steam/steam/logs/steamui_html.txt
/home/inkrm/.steam/steam/logs/connection_log.txt
/home/inkrm/.steam/steam/logs/steamui_update.txt
/home/inkrm/.steam/steam/logs/stats_log.txt
/home/inkrm/.steam/steam/logs/cef_log.previous.txt
```

#### Please describe your issue in as much detail as possible: I have a 28 inch 4K monitor and Steam doesn't scale it's window at all
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/): I expected to just turn on 'scale text and icons to match monitor settings' in settings -> interface, but it is turned on and still very tiny

#### Steps for reproducing this issue:
       
1. Open Steam
2. Go to up left corner and click on Steam, then click on settings -> interface
3. See the 'Scale text and icons' turned on

**Comment by kisak-valve** [comment]
Hello @Inkrm, this issue is already being tracked at #9209.

You should be able to workaround this issue by completely closing Steam, then running Steam with `STEAM_FORCE_DESKTOPUI_SCALING=2.0 steam` or `steam -forcedesktopscaling=2.0` (adjusting 2.0 to your preference).

Closing in favor of the older issue report.

**Comment by MartinVonReichenberg** [comment]
Confirming it here too:
Seems like Ubuntu and Arch Linux are not aware of **fractional** (150%) **scaling** on X-org **X11** GNOME session:
On SUSE it works _out-of-the-box_.

```
martin@Kingston-USB-Flash-Drive--Ubuntu-23-10 ~> inxi -F
System:
  Host: Kingston-USB-Flash-Drive--Ubuntu-23-10 Kernel: 6.5.0-9-generic
    arch: x86_64 bits: 64 Desktop: GNOME v: 45.0 Distro: Ubuntu 23.10 (Mantic
    Minotaur)
Machine:
  Type: Laptop System: LENOVO product: 82JQ v: Legion 5 Pro 16ACH6H
    serial: <superuser required>
  Mobo: LENOVO model: LNVNB161216 v: NO DPK serial: <superuser required>
    UEFI: LENOVO v: GKCN60WW date: 03/07/2023
Battery:
  ID-1: BAT0 charge: 83.7 Wh (100.0%) condition: 83.7/80.0 Wh (104.6%)
CPU:
  Info: 6-core model: AMD Ryzen 5 5600H with Radeon Graphics bits: 64
    type: MT MCP cache: L2: 3 MiB
  Speed (MHz): avg: 1881 min/max: 400/4280 cores: 1: 2716 2: 2633 3: 2229
    4: 400 5: 2205 6: 2516 7: 2128 8: 400 9: 400 10: 2396 11: 2530 12: 2020
Graphics:
  Device-1: NVIDIA GA106M [GeForce RTX 3060 Mobile / Max-Q] driver: nvidia
    v: 535.113.01
  Display: x11 server: X.Org v: 1.21.1.7 with: Xwayland v: 23.2.0 driver: X:
    loaded: nvidia unloaded: fbdev,modesetting,nouveau,vesa
    gpu: nvidia,nvidia-nvswitch resolution: 3408x2130
  API: OpenGL v: 4.6.0 NVIDIA 535.113.01 renderer: NVIDIA GeForce RTX 3060
    Laptop GPU/PCIe/SSE2
Audio:
  Device-1: NVIDIA GA106 High Definition Audio driver: snd_hda_intel
  Device-2: AMD ACP/ACP3X/ACP6x Audio Coprocessor driver: N/A
  Device-3: AMD Family 17h/19h HD Audio driver: snd_hda_intel
  API: ALSA v: k6.5.0-9-generic status: kernel-api
  Server-1: PipeWire v: 0.3.79 status: active
Network:
  Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet
    driver: r8169
  IF: eno1 state: up speed: 1000 Mbps duplex: full mac: 9c:2d:cd:c9:f6:9b
  Device-2: MEDIATEK MT7921 802.11ax PCI Express Wireless Network Adapter
    driver: mt7921e
  IF: wlp4s0 state: down mac: f0:a6:54:33:f4:2d
  Device-3: Realtek RTL8153 Gigabit Ethernet Adapter driver: r8152 type: USB
  IF: enx9405bb1970c2 state: down mac: 94:05:bb:19:70:c2
Bluetooth:
  Device-1: Foxconn / Hon Hai MediaTek Bluetooth Adapter driver: btusb
    type: USB
  Report: hciconfig ID: hci0 rfk-id: 2 state: down
    bt-service: enabled,running rfk-block: hardware: no software: yes
    address: F0:A6:54:33:F4:2E
Drives:
  Local Storage: total: 505.77 GiB used: 22.83 GiB (4.5%)
  ID-1: /dev/nvme0n1 vendor: Samsung model: MZVLB512HBJQ-000L2
    size: 476.94 GiB
  ID-2: /dev/sda vendor: Kingston model: DataTraveler 3.0 size: 28.83 GiB
    type: USB
Partition:
  ID-1: / size: 27.17 GiB used: 22.82 GiB (84.0%) fs: ext4 dev: /dev/sda2
  ID-2: /boot/efi size: 1.05 GiB used: 6.1 MiB (0.6%) fs: vfat
    dev: /dev/sda1
Swap:
  ID-1: swap-1 type: file size: 4 GiB used: 0 KiB (0.0%) file: /swap.img
Sensors:
  System Temperatures: cpu: 42.2 C mobo: N/A gpu: nvidia temp: 39 C
  Fan Speeds (rpm): N/A
Info:
  Processes: 362 Uptime: 1h 0m Memory: total: 16 GiB available: 15.46 GiB
  used: 3.62 GiB (23.4%) Shell: fish inxi: 3.3.29
```

#### More shown in Pictures:
![Screenshot from 2023-10-24 16-24-26](https://github.com/canonical/steam-snap/assets/110736521/15ab5595-f4d6-4984-8d13-6a83434da358)
![Screenshot from 2023-10-24 16-32-09](https://github.com/canonical/steam-snap/assets/110736521/01fa0f84-a30f-406a-87e7-69eeb42a84a1)
![Screenshot from 2023-10-24 16-33-12](https://github.com/canonical/steam-snap/assets/110736521/c0cecbef-2d29-4610-91d6-50a907e11789)
![Screenshot from 2023-10-24 16-46-54](https://github.com/canonical/steam-snap/assets/110736521/64481076-62b3-4beb-b5e9-b5a6eda03eb3)

TL;DR:
**_There only two ways of enabling the Steam's fractional scaling along with the rest of the system:
Using `steam -forcedesktopscaling 1.5 %U` ARGUMENT
or
`env GDK_SCALE=2 steam` / `export GDK_SCALE=2 & steam`_**

**_They can be added to their respective application launchers in:
`/var/lib/snapd/desktop/applications/steam_steam.desktop` for the Snap version
`/usr/share/applications/steam.desktop` for the .DEBian version
`~/.local/share/applications/com.valvesoftware.Steam.desktop` for the Flatpak version_**

**Comment by Inkrm** [comment]
@MartinVonReichenberg thank you very much
